src/wasm_worker.js

```diff
@@ -1,10 +1,38 @@
+/**
+ * Called once the intiial message has been recieved from the creating thread.
+ * The `props` object is the list of properties sent via postMessage to create
+ * the worker.
+ *
+ * This function is called both in normal wasm workers and in audio worklets.
+ */
+function startWasmWorker(props) {
+#if RUNTIME_DEBUG
+  dbg('startWasmWorker');
+#endif
+#if MINIMAL_RUNTIME
+  Module ||= {};
+#endif
+  /** @suppress {checkTypes} */
+  Object.assign(Module, props);
+  wasmMemory = props['mem'];
+  updateMemoryViews();
+#if MINIMAL_RUNTIME
+  loadModule()
+#else
+  wasmModuleReceived(props['wasm']);
+#endif
+  // Drop now unneeded references to from the Module object in this Worker,
+  // these are not needed anymore.
+  props['wasm'] = props['mem'] = 0;
```

Comment:
You mentioned in the previous PR:

> Once we do the followup of also inlining the AUDIO_WORKLET file I think we can completly remove these quotes and let closure minify these names (like we did in the pthread case) which will further reduce code size.

Should we do this now or left as a follow-up?

Reply:
Lets do that as a followup yes.